src/store/modules/alerts.store.ts

```diff
@@ -58,10 +58,11 @@ const mutations = {
     state.isSearching = true
     state.query = query
   },
-  SET_ALERTS(state, alerts): any {
+  SET_ALERTS(state, result): any {
     state.isLoading = false
     state.isSearching = false
-    state.alerts = alerts
+    state.alerts = result.alerts
+    state.pagination.totalItems = result.total
```

Comment:
This could be `alerts.length` then nothing else has to change.

Reply:
alerts.length is the count in current page, but result.total is total count of alerts.